packages/expo/src/logs/Logs.ts

```diff
@@ -1,21 +1,24 @@
-import RemoteConsole from './RemoteConsole';
-
 let _originalConsole: typeof console | null;
 
 export function enableExpoCliLogging(): void {
-  if (_originalConsole) {
-    return;
-  }
+  if (__DEV__) {
```

Comment:
Is this necessary with the main DEV block?

Reply:
yes, the other dev block is a side effect, this dev block is for the re-export through the main expo module.